modules/PersonalBank.py

```diff
@@ -14,10 +14,12 @@ class PersonalBank:
     """
 
     
-    def __init__(self, balance, history_of_transactions):      
-    
-        self.balance = balance
-        self.history_of_transactions = history_of_transactions
+    def __init__(self, balance = None):      
+        
+        if balance == None:
+            self.balance = 350
+        else:
+            self.balance = balance
 
     def add_transaction(history_of_transactions, transaction):
 
```

Comment:
This method should add a transaction to the history. It's a single line `self.history_of_transactions.append(transaction)`.

Reply:
The method `add_transaction()` shouldn't have the `history_of_transactions` as an attribute. It should just append a `transaction` directly to the history. So the `history_of_transactions` should be defined somewhere in the class but not in the constructor as an instance variable